saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -114,9 +122,19 @@ def clean_voucher(cls, voucher, channel):
                 }
             )
 
+    @classmethod
+    def clean_redirect_url(cls, redirect_url):
+        try:
+            validate_storefront_url(redirect_url)
+        except ValidationError as error:
+            raise ValidationError(
+                {"redirect_url": error}, code=OrderErrorCode.INVALID.value
+            )
```

Comment:
I know that it's done like this everywhere, but error code should be attached a level higher to the `error` object like:
```suggestion
            error.code = OrderErrorCode.INVALID.value
            raise ValidationError(
                {"redirect_url": error}
             )
```